saw-script.cabal

```diff
@@ -1,6 +1,6 @@
 Cabal-version: 2.4
 Name:          saw-script
-Version:       1.1.0.99
+Version:       1.2.0
```

Comment:
Similarly with Crux, we've used the convention of writing `1.2` instead of `1.2.0` up until now, so we may as well stick with that. (Similarly in the commit message.)